proposals/0029-cooperative-vector.md

```diff
@@ -546,7 +546,8 @@ typedef enum D3D12_COOPERATIVE_VECTOR_DATATYPE {
 typedef enum D3D12_COOPERATIVE_VECTOR_TIER
 {
     D3D12_COOPERATIVE_VECTOR_TIER_NOT_SUPPORTED,    
-    D3D12_COOPERATIVE_VECTOR_TIER_1_0
+    D3D12_COOPERATIVE_VECTOR_TIER_1_0,
```

Comment:
If we rename COOPERATIVE_VECTOR to something like LINEAR_ALGEBRA, would this tiering system still make sense, in that mat*mat would layer on top?  Or would we need indepdendent tier progressions for vecMat vs matMat?

If we need separate tier progresssions, could the names be D3D12_LINEAR_ALGEBRA_VEC_MAT_TIER and (future) D3D12_LINEAR_ALGEBRA_MAT_MAT_TIER?  Or is there a more accurate naming?

Reply:
We could do such a rename in a different PR, just trying to understand the implications for this tiering.

Reply:
I think D3D12_LINEAR_ALGEBRA_**2**_* for Mat-Mat operations, leaving D3D12_LINEAR_ALGEBRA_1_* should be sufficient?

Reply:
I think more discussion here is required, but it outside the scope of this PR so I'm going to go ahead and merge this one.